smart-contracts/osmosis/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -105,40 +99,30 @@ pub fn execute_swap_non_vault_funds(
 pub fn calculate_swap_amount(
     contract_address: Addr,
     pool_config: PoolConfig,
-    swap_direction: SwapDirection,
-    token_in_amount: Uint128,
+    offer: Coin,
```

Comment:
Is a bit confusing the way we call tokens, which sometimes is token_in and here is offer.

Reply:
done